src/parser/tests/expression/AssociativeArrayLiterals.spec.ts

```diff
@@ -205,6 +207,68 @@ describe('parser associative array literals', () => {
         ]);
     });
 
+    describe('unfinished', () => {
+        it('will still be parsed', () => {
+            // No closing brace:  _ = {name: "john", age: 42, address: data.address
+            let { statements, diagnostics } = Parser.parse([
```

Comment:
These verbose tests are just a holdover from the `brs` project. You can convert this test into a standard template string instead of building the whole array of tokens.

Reply:
fixed